### DISCUSSION TITLE
Recurring meetings

### DISCUSSION THREAD

**Discussion by astrojuanlu** [lukevella/rallly#163] [Ideas]
Just discovered Rally, the UI is so sleek ✨ Kudos!

What I actually wanted though is a tool for recurring weekly meetings, hence not in specific dates. Something like https://www.when2meet.com, but with the beauty of Rallly. This means:

- Table view, i.e. rows are time slots, and columns are days of the week (otherwise, the list of options becomes way too long and the user needs to do lots of horizontal scrolling), and
- Generic days of the week, not attached to a particular calendar day

I don't think there are many tools out there that do this nicely, it would be so cool for Rallly to support this.

**Comment by lukevella** [comment]
Thanks @astrojuanlu appreciate the kind words. I will try and take Rallly in to this space soon as well but maybe check out [crab.fit](https://crab.fit) as well if you haven't already as it does exactly that with a more polished UI.

**Reply by astrojuanlu** [reply]
Yep, I do know it :) https://github.com/GRA0007/crab.fit/issues/138

**Comment by adan-ea** [comment]
Hello there, wanted to know if there was anything new on this. I love rallly and it would be such a great addition to receive a mail every weekend with a recurring meeting 
edit: no pressure, just curious about it! I'll accept not having an answer as an answer
